androidApp/home/src/main/java/tech/takahana/iconwallpaper/android/home/ui/components/ImagePattern.kt

```diff
@@ -0,0 +1,58 @@
+package tech.takahana.iconwallpaper.android.home.ui.components
+
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.res.imageResource
+import androidx.compose.ui.tooling.preview.Preview
+import tech.takahana.iconwallpaper.android.core.ui.theme.IconWallPaperTheme
+import tech.takahana.iconwallpaper.android.home.R
+import tech.takahana.iconwallpaper.android.home.ui.util.DrawScopeUtils.drawPattern
+import tech.takahana.iconwallpaper.shared.domain.domainobject.PatternType
+
+@Composable
+fun ImagePattern(patternType: PatternType) {
+    val image = ImageBitmap.imageResource(
+        id = R.drawable.cat
+    )
+    val backGroundColor = MaterialTheme.colors.background
```

Comment:
細かいですが、 `MaterialTheme.colors.background` に合わせてGを小文字にしてもいいかも👀

```suggestion
    val backgroundColor = MaterialTheme.colors.background
```

Reply:
こちらのコミットで変更しました！
585aada6d1d7ddd9d2eb8348ef939f836e4e8630